tools/scripts/tt

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python3
+"""tt - the xfiles-dana test tool
+
+Provides functions for running emulator tests and inspecting memory
+
+Usage: tt --help
+"""
+
+import click
+from re import compile
+from os import path
+from subprocess import run, PIPE, DEVNULL, TimeoutExpired
+# from ipdb import set_trace
+from os import system
+
+
+@click.group()
+@click.option('--debug', is_flag=True)
+@click.option('--rocket_chip_project_path')
+@click.pass_context
+def tt(ctx, debug, rocket_chip_project_path):
+    ctx.obj['DEBUG'] = debug
+    rocket_chip_project_path = rocket_chip_project_path or os.environ.get(
```

Comment:
I was running this on my machine and I needed to `import os`. Can you verify that this does work?